pkgs/top-level/all-packages.nix

```diff
@@ -21369,7 +21385,9 @@ with pkgs;
   libGLU = mesa_glu;
 
   mesa = callPackage ../development/libraries/mesa {
-    llvmPackages = llvmPackages_latest;
+    # LLVM 15 isn't supported until MESA 22.2; use LLVM 14 for now:
```

Comment:
`master` is on 22.2.5, so this should be fine now? And FWIU, mesa w/ LLVM 15 is needed for the newly-released RDNA3.

Reply:
Yep RDNA3 does hard-require LLVM15 (from personal hacking on #204125)
Though currently llvmPackages_git is good enough to get away it feels awfully hacky